ADALiOS/ADALiOS/NSString+ADExtentions.m

```diff
@@ -0,0 +1,45 @@
+//
+//  NSString+ADExtentions.m
+//  ADALiOS
+//
+//  Created by Brandon Werner on 9/11/15.
+//  Copyright © 2015 MS Open Tech. All rights reserved.
+//
+
+#import "NSString+ADExtentions.h"
+
+@implementation NSString (ADExtensions)
+
+- (NSString*)adSpaceDeliminatedString
```

Comment:
I was pointing you you could replace that whole implementation with one line :). I'm fine with keeping the category method around, especially cause it might allow us to be lazier in other contexts... sorry I mean "more dynamic."